packages/contracts-bedrock/test/kontrol/scripts/json/clean_json.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import json
+
+def clean_json(input_file):
```

Comment:
I am curious if this can all be accomplished with `jq`

Reply:
I've added an issue in our fork to keep this live (https://github.com/runtimeverification/_audits_Ethereum-optimism_pausability/issues/4). I'll go ahead and resolve this conversation then. Let me know if you have any further comments!